src/coord/scaleRawExtentInfo.ts

```diff
@@ -234,6 +234,20 @@ export class ScaleRawExtentInfo {
             maxFixed = true;
         }
 
+        // Use this where only one of the min or max is fixed and the other not.
+        // You can see the usage of this when testing min-max-axis.html that I created as a test case.
+        // (https://github.com/apache/echarts/issues/19187)
+        if ((!maxFixed && minFixed) || (!minFixed && maxFixed)) {
+            if (min > max) {
+                max = min + 10;
```

Comment:
What does the magic number 10 mean here?

Reply:
@Ovilia, thank you for your response.

My reasoning behind the 10 offset was since the user won't supposed to see any data on the graph with a min that does not get override, the user should still see a range of numbers for the labels. 

However, I can change the set magical number logic or set it to something else. I would be happy if I get your further feedback on the topic. Thanks!

Reply:
Typically, we provide an option like 'xAxis.xxx' for this value. Thanks!

Reply:
If I remove the magic number, y axis range becomes 2.5 as min and 5 as max as in the example below. With this, it does not show the data that is not in the ymin range as it should, however, it still ignores the min threshold that the user set (ymin = 5 in this example below). 

Would removing the magic number fix the issue or what do you suggest?

Thanks!

<img width="793" alt="Screenshot 2024-01-05 at 7 44 33 PM" src="https://github.com/apache/echarts/assets/31061207/ab4c3846-c7f0-4cb0-83df-88c526ae69b8">